action.yml

```diff
@@ -99,6 +99,7 @@ runs:
                 createNamespace: true
                 skipBuildDependencies: true
                 upgradeOnChange: true
+                wait: true
```

Comment:
As I remember, when [I added skaffold](https://github.com/cloudbees-io/helm-install/pull/2) in order to log deployment errors, initially I also specified the `wait: true` option but unfortunately that resulted in skaffold not polling the deployment status and logging detailed errors. I had to turn off the `wait` option in order to let skaffold wait for the deployment to succeed instead of letting Helm do that and not getting detailed info about rollout errors.
That said, since I added skaffold this way, we're seeing deployment errors logged properly and skaffold is waiting for the deployments to succeed and otherwise it fails as required. Are you sure enabling the `wait` option now lets skaffold still log deployment errors? (Maybe skaffold changed that behaviour within the last update we applied?)

Reply:
you are correct, the behavior remains un-changed. Will upgrade the version of `skaffold` and see if that fixes this issue.

Reply:
As per @mgoltzsche suggestion, switched to using `tolerateFailuresUntilDeadline` instead of `wait`